corehq/apps/app_manager/views/utils.py

```diff
@@ -289,6 +290,22 @@ def handle_custom_icon_edits(request, form_or_module, lang):
             form_or_module.custom_icons = []
 
 
+@task(serializer='pickle', queue='background_queue', ignore_result=True)
+def update_linked_app_and_notify(domain, app_id, user_id, email):
+    app = get_current_app(domain, app_id)
+    try:
+        update_linked_app(app, user_id)
+    except AppLinkError as e:
+        message = str(e)
```

Comment:
This should be `six.text_type` or something for Python3 compatability

Reply:
thanks, updated.